android/src/main/java/com/oney/WebRTCModule/GetUserMediaImpl.java

```diff
@@ -59,6 +77,24 @@
             Log.d(TAG, "Creating video capturer using Camera1 API.");
             cameraEnumerator = new Camera1Enumerator(false);
         }
+
+        reactContext.addActivityEventListener(new BaseActivityEventListener() {
+            @RequiresApi(api = Build.VERSION_CODES.N)
+            @Override
+            public void onActivityResult(Activity activity, int requestCode, int resultCode, Intent data) {
+                super.onActivityResult(activity, requestCode, resultCode, data);
+                if (requestCode == PERMISSION_REQUEST_CODE) {
+                    if (resultCode != Activity.RESULT_OK) {
+                        promise.reject("DOMException", "NotAllowedError");
+                        promise = null;
```

Comment:
Is it necessary to nullify the promise here?

Reply:
It's necessary for the logic in getDisplayMedia(Promise promise) to work, where we verify that there isn't another task in progress.